### PR TITLE
[9.0][mig] fetchmail_bydate

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Migration to from 7.0 9.0.

Fetchmail Bydate
================

This module allows to fetch new emails (using IMAP) received from the last
time they were downloaded and successfully processed, in addition to 'unseen'
status.

Users with authorization to edit the email server in Odoo can introduce a
new date and time to download from.

In case of errors found during the processing of an email Odoo will
re-attempt to fetch the emails from the last date and time they were
successfully received and processed.

Configuration
=============

To enable this, you have to set a 'Last Download Date' in the fetchmail.server
After that, emails with an internal date greater than the saved one will be
downloaded.

Usage
=====

Odoo will attempt to fetch emails starting from the 'Last Download Date'
defined in the email server. If all mails have been processed successfully,
it will update this date with the latest message received.

System administrators need to be attentive to the Odoo logs, looking for errors
raised during the processing of emails, in order to avoid situations
lots of emails are now being downloaded and reprocessed every time, due to
errors found in a few old emails.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
You might be interested in this one @eLBati @archetipo @aheficent

File: fetchmail_bydate/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+=============
+
+To enable this, you have to set a 'Last Download Date' in the fetchmail.server
+After that, emails with an internal date greater than the saved one will be
+downloaded.
+
+
+.. figure:: path/to/local/image.png
+   :alt: alternative description
+   :width: 600 px
```
Comment (yajo) [inline fetchmail_bydate/README.rst]: 🔥

Comment (JordiBForgeFlow) [reply fetchmail_bydate/README.rst]: oops. Fixed.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  Functional test only

**Pull request merged**